src/odm_sharing/private/trees.py

```diff
@@ -485,11 +485,18 @@ def parse(rules: Union[Dict[RuleId, Rule], List[Rule]],
 
     :return: an opaque rule-tree object for query generation
     '''
+    ctx = Ctx(filename)
+
     if isinstance(rules, dict):
         rules = list(rules.values())
     org_whitelist = set(orgs)
+    orgs_in_schema = seq(rules)\
+        .filter(lambda r: r.mode == RuleMode.SHARE)\
+        .map(lambda r: r.key)\
+        .set()
 
-    ctx = Ctx(filename)
+    if not (org_whitelist <= orgs_in_schema):
```

Comment:
Is there a reason you're not using a greater than here?

Reply:
This doesn't check if an org is missing right? For example if the schema has only `OHRI` but the orgs whitelist is `StatCan`?

Reply:
i'm using less-than because i'm checking if "the whitelist is a subset of the orgs in the schema", and if `not`, then branch.

StatCan is not a subset of OHRI, so that would fail

Reply:
Can we not check if the `org_whitelist` is a superset of `orgs_in_schema`, using the greater then operator? or is that not equivalent to `not <=`.

Reply:
sure, I just think it's less intuitive. I wrote it in the way that's easy to reason about, imo.